### PR TITLE
Extend pathway map uploader process

### PR DESCRIPTION
**Pull request by matyasfodor**
~ with species and source field

### CODE REVIEW

File: resolwe_bio/processes/import_data/pathway_map.yml [hunk 1]
```diff
@@ -22,13 +22,31 @@
...
+    - name: species
+      label: Species
+      type: basic:string
   run:
     runtime: polyglot
     language: bash
     program: |
       re-import {{ src.file_temp }} {{ src.file }} "json" "json" 0.5
       re-save pathway "${NAME}.json"
+      re-save source {{source}}
```
Comment (jkokosar) [inline resolwe_bio/processes/import_data/pathway_map.yml]: `{{ source }}` sytax is preferred.

File: resolwe_bio/processes/import_data/pathway_map.yml [hunk 2]
```diff
@@ -22,13 +22,31 @@
...
         JSON file. Supported extensions: '.json', '.json.gz'
       validate_regex: '(\.json)(\.gz)?$'
+    - name: source
+      label: source
+      type: basic:string
+      allow_custom_choice: true
+      choices:
+        - label: BIGG
+          value: BIGG
+    - name: species
```
Comment (jkokosar) [inline resolwe_bio/processes/import_data/pathway_map.yml]: Add `choices` menu with the list of supported species.

File: resolwe_bio/processes/import_data/pathway_map.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@
     resources:
       network: true
   data_name: Metabolic Pathway
-  version: 0.0.1
+  version: 0.1.2
```
Comment (jkokosar) [inline resolwe_bio/processes/import_data/pathway_map.yml]: bump to either 0.0.2 or 0.1.0

**Comment by matyasfodor** [comment]
retest this please

**Pull request merged**